### PR TITLE
Des 160 create the cambiar contrasena option from profile page phase 2 front end

### PR DESCRIPTION
**Pull request by Daniielmayo**

### CODE REVIEW

**Review by abdulflorez — APPROVED** [review]
Looks good to me

**Review by DanielCalleCO — CHANGES_REQUESTED** [review]

File: src/components/organims/ChangePasswordProfile/ChangePasswordProfile.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  /* animation-name: modal-closed; */
+  animation-duration: 0.5s;
+}
+/* Animación de entrada */
```
[the remark this answers is not in the document]
Comment (Daniielmayo) [reply src/components/organims/ChangePasswordProfile/ChangePasswordProfile.css:25]: this line is to delete

File: src/components/organims/ChangePasswordProfile/ChangePasswordProfile.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+.input-modal {
+  padding: 0.6rem;
+  border-radius: 0.8rem;
+  width: 15rem;
+  border: none;
+  outline: 0.05rem solid var(--turquoise);
+  background-color: transparent;
+  color: black;
+  font-size: 1rem;
+  margin-bottom: 16px;
```
[the remark this answers is not in the document]
Comment (Daniielmayo) [reply src/components/organims/ChangePasswordProfile/ChangePasswordProfile.css:51]: change of px to rem

File: src/components/organims/ChangePasswordProfile/ChangePasswordProfile.css [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  }
+  .input-modal {
+    border-radius: 0.8rem;
+    width: 25rem;
+    font-size: 1rem;
+  }
+}
+@media (min-width: 64rem) {
+  .input-modal {
+    /* padding:1.5rem; */
```
[the remark this answers is not in the document]
Comment (Daniielmayo) [reply src/components/organims/ChangePasswordProfile/ChangePasswordProfile.css:89]: deleted

File: src/components/organims/Profile/Profile.jsx [hunk 4]
```diff
@@ -230,6 +238,17 @@ export const Profile = () => {
               >
                 Editar perfil
               </button>
+              <buttom
+                className="profile-user_changeInfo_btn"
+                onClick={() => navigateChangePassword()}
+              >
+                Cambiar contraseña
+              </buttom>
+              {/* <ChangePasswordProfile
```
[the remark this answers is not in the document]
Comment (Daniielmayo) [reply src/components/organims/Profile/Profile.jsx:247]: deleted comment

**Review by DanielCalleCO — APPROVED** [review]
Awesome JOB!! thanks for the changes Dani

**Pull request merged**